data/cameras.xml

```diff
@@ -7387,6 +7387,42 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
+	<Camera make="OM Digital Solutions" model="OM-1">
+		<ID make="OM System" model="OM-1">OM Digital Solutions OM-1</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">RED</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">BLUE</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
+		<Sensor black="254" white="4000"/>
+		<ColorMatrices>
+			<ColorMatrix planes="3">
+				<ColorMatrixRow plane="0">9488 -3984 -714</ColorMatrixRow>
+				<ColorMatrixRow plane="1">-2887 10945 2229</ColorMatrixRow>
+				<ColorMatrixRow plane="2">-137 960 5786</ColorMatrixRow>
+			</ColorMatrix>
+		</ColorMatrices>
+	</Camera>
+	<Camera make="OM Digital Solutions" model="OM-5">
```

Comment:
Perhaps add a comment these are copied from Olympus E-M5 Mark III?

Reply:
Actually, this is the output of `tools/dngmeta.sh`, run on an image from OM-5. Except for make and model it happens to be identical to that of E-M5 mkIII.

Reply:
As there is no official support listed by ADC for this model, I guess either 1) wait until it is listed (probably next version) for this method, or 2) mention the copy route.

Reply:
d4311a24 reverted. Perhaps it's better to add OM-5 in a separate simpler step after general support for OM System has been merged.

Reply:
P.S. Does ADC version go in sync with that of Camera Raw? If yes, v.15 does support OM-5.
https://helpx.adobe.com/camera-raw/kb/camera-raw-plug-supported-cameras.html

Reply:
> P.S. Does ADC version go in sync with that of Camera Raw? If yes, v.15 does support OM-5. https://helpx.adobe.com/camera-raw/kb/camera-raw-plug-supported-cameras.html

Should do indeed, looks like the UK site is not up to date!? https://helpx.adobe.com/uk/camera-raw/kb/camera-raw-plug-supported-cameras.html Go figure...

Reply:
> In any case, unlikely to be accepted by the maintainer as there is no sample at RPU, so might as well split into a separate PR...

There's OM-5 samples at RPU! (https://github.com/darktable-org/rawspeed/pull/399#issuecomment-1317613700)